src/cores.tsx

```diff
@@ -0,0 +1,36 @@
+const coresBrand = {
+  primary: {
+    Dark: "#031F3D",
+    Medium: "#0C4888",
+  },
+  ctaMedium: {
+    cta: "#29D6E7",
+  },
+  acessory: {
+    backgroundSecundario: "#0C4888",
+    backgroundBanner: "#28ADBA",
+  },
+
+};
+const neutral = {
+  white: "#FFFFFF",
+  neutralXLight: "#FDFDFD",
+  neutralLight: "#C5C5C5",
+  neutralMedium: "#908E8E",
+  neutralDark: "#3A3335",
+  neutralXDark: "#484848",
+};
+
+const feedback = {
+  FeedbackSuccess: "#2AC084",
+  FeedbackError: "#EB5E55",
+};
+
+export { }
```

Comment:
Aqui você acabou não exportando nada no arquivo, então quem for usar de fora não vai ter o que importar. Uma dica seria vc exportar um objeto que tem um campo pra cada variável que vc criou ali em cima, assim ele exporta as cores divididas nos grupos que vc dividiu:

```ts
export {
  brandColors,
  neutral,
  feedback,
}
```